config/deploy.rb

```diff
@@ -1,6 +1,6 @@
 # Change these
-set :repo_url,        'git@github.com:rakvium/sloboda-supersonic-taxi.git'
-set :application,     'sloboda-supersonic-taxi'
+set :repo_url,        'git@github.com:Kolisnyk/test.git'
```

Comment:
I've been neglectful, and I'm sorry. It's wrong version. I'm gonna to made new commit.